quickwit-config/resources/tests/server_config/server.toml

```diff
@@ -0,0 +1,15 @@
+metastore_uri = "postgres://username:password@host:port/db"
+
+[indexer]
+data_dir_path = "/opt/quickwit/data"
+split_store_max_num_bytes = "100G"
+split_store_max_num_files = 1_000
+
+[searcher]
+data_dir_path = "/opt/quickwit/data"
+peer_seeds = [ "quickwit-searcher-0.local", "quickwit-searcher-1.local" ]
+fast_field_cache_capacity = "1G"
```

Comment:
there is also a `split_footer_cache_capacity`. We may call it `hotcache_cache_capacity` or `split_turbo_cache_capacity` any cool name you can think of :)

Reply:
it's right below :D